cli/objects/rule.py

```diff
@@ -262,37 +249,34 @@ def _get_jira_additional_labels(
             Optional[list[str]]: A list of strings representing additional labels.
         """
         labels = []
-
-        if "jira_additional_labels" in rule_dict.keys():
-
-            jira_additional_labels = rule_dict["jira_additional_labels"]
-
-            if isinstance(jira_additional_labels, str):
-                labels.append(jira_additional_labels)
-                return labels
-            elif isinstance(jira_additional_labels, list):
-                for label in jira_additional_labels:
-                    if isinstance(label, str):
-                        if " " in label:
-                            self.logger.error(
-                                f'Label "{label}" in rule {rule_dict} contains spaces. Remove spaces and try again.',
-                            )
-                            exit(1)
-                        else:
-                            labels.append(label)
-                    else:
+        jira_additional_labels = rule_dict.get("jira_additional_labels")
+
+        if isinstance(jira_additional_labels, str):
```

Comment:
ditto, I think it's better to always get the same type (list in this case)
and if you only get list from the user then this function can be deleted.

Reply:
This one we can change because it is fairly new and the documentation has always reflected that this should be a list of strings. I believe I just used the method from jira_component and forgot to remove that part.